crates/loader/src/local/assets.rs

```diff
@@ -260,3 +289,36 @@ fn is_absolute_guest_path(path: impl AsRef<Path>) -> bool {
     // paths specifically using Unix logic rather than the system function.
     path.as_ref().to_string_lossy().starts_with('/')
 }
+
+/// Convert strings to glob patterns
+pub fn convert_strings_to_glob_patterns<T: AsRef<str>>(files: &[T]) -> Result<Vec<glob::Pattern>> {
+    files
+        .iter()
+        .map(|f| {
+            glob::Pattern::new(f.as_ref())
+                .with_context(|| format!("can't convert {} to glob pattern", f.as_ref()))
+        })
+        .collect::<Result<Vec<glob::Pattern>>>()
+}
+
+/// Return whether the file path in in excluded patterns
+pub fn is_excluded_file(
```

Comment:
Does this need to be public?  It feels like nothing outside the loader should care about it...?

Reply:
I remove the function. Thank you.